doc/dev/ai_prompt_workflow.md

```diff
@@ -0,0 +1,14 @@
+# AI Prompt Workflow
+
+
+1. In Copilot **Agent** mode, click on `add_context` and select `prompt` from the dropdown.
+
+*Note*: Agent mode is currently only available in VSCode Insiders.
+
+![Agent Mode](step1.png)
+
+2. Select `next-pylint.prompt.md` or associated prompt for your task.
+
+![Select Prompt](step2.png)
+
+3. Send the prompt to the agent. Follow the instructions provided by the agent to complete the task.
```

Comment:
when you say "send the prompt" to the agent, does that mean have it show up in the context?